src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/HologramProjector.java

```diff
@@ -42,7 +53,9 @@ private BlockPlaceHandler onPlace() {
             public void onPlayerPlace(BlockPlaceEvent e) {
                 Block b = e.getBlockPlaced();
                 BlockStorage.addBlockInfo(b, "text", "Edit me via the Projector");
-                BlockStorage.addBlockInfo(b, OFFSET_PARAMETER, "0.5");
+                BlockStorage.addBlockInfo(b, X_OFFSET_PARAMETER, "0.5");
+                BlockStorage.addBlockInfo(b, Y_OFFSET_PARAMETER, "0.5");
+                BlockStorage.addBlockInfo(b, Z_OFFSET_PARAMETER, "0.5");
```

Comment:
You are using "0.5" a freaking lot here, I think you should make a final field for the default values here and just read from that...
If we ever wanna change these, we'd need to change them in a thousand places. This is not feasible.